src/SdkCommon/ClientRuntime.Azure/ClientRuntime.Azure/CloudError.cs

```diff
@@ -37,5 +38,11 @@ public CloudError()
         /// Gets or sets details for the error.
         /// </summary>
         public IList<CloudError> Details { get; private set; }
+
+        /// <summary>
+        /// Gets or sets additional error info.
+        /// </summary>
+        [JsonProperty(ItemConverterType = typeof(TypedErrorInfoJsonConverter))]
+        public IList<TypedErrorInfo> AdditionalInfo { get; set; }
```

Comment:
@j5lim this change has a potential to introduce runtime exceptions for legacy AP versions for services.
This change will need to thoroughly tested for all services (including past API versions)

Reply:
I think the tests already cover that case (the response missing additionalInfo property, which is optional). Do you think it is not enough?